layouts/black-lives-matter.hbs

```diff
@@ -0,0 +1,229 @@
+      <li><a href="https://www.facebook.com/reclaimtheblock/">Reclaim the Block</a> has published a <a href="https://docs.google.com/document/d/1yLWGTQIe3967hdc9RSxBq5s6KKZHe-3_mWp5oemd7OA/preview?pru=AAABcpUiX3k*Y6Q4I6UBtkH3lLz9GVLg0A">Google Doc of organizations to donate to.</a></li>
+    </ul>
+    <p>We will be updating this list periodically.</p>
+  </section>
+
+  <section id="our-support">
+
+  <p>In Node.js, we recognize that there is still an incredible amount of work to be accomplished to address the injustices and inequalities that exist, not only in society as a whole but within the technology industry, and even within our own project. To this point, we have not done enough and recognize that we need to do more. We commit to supporting the Black members of the community by using our collective platforms to amplify your words, or by helping you in your careers, contributing to open source, and seeking other opportunities to help.</p>
+
+ <p>Beginning with our upcoming Collaborator Summit event, and extending beyond, we will be offering mentorship to Black members of the community who wish to become contributors to the Node.js project. If you are interested in this, please send us an email at <a href="mailto:blacklivesmatter@nodejs.org">blacklivesmatter@nodejs.org</a> or direct message us at <a href="https://twitter.com/nodejs">@nodejs</a> on Twitter. We will post more information on how we will be proceeding with this effort as we finalize the details both for the period of the Collaborator summit and beyond that single event.</p>
+
+    <section id="we-are-here-for-you">
+      <p>Please feel welcome to reach out to the <a href="https://github.com/nodejs/node#tsc-technical-steering-committee">Node.js TSC members</a> and <a href="https://github.com/nodejs/community-committee#community-committee-members">Node.js Community Committee members</a> via our email addresses listed in the links, respectively, or via Twitter DMs to the following folks in the project:
+      <ul>
+        <li><a href='https://twitter.com/bitandbang'>Tierney Cyren</a></li>
+        <li><a href='https://twitter.com/joe_sepi'>Joe Sepi</a></li>
+        <li><a href='https://twitter.com/jasnell'>James M Snell</a></li>
+        <li><a href='https://twitter.com/MylesBorins'>Myles Borins</a></li>
+        <li><a href='https://twitter.com/mmarkini'>Matheus Marchini</a></li>
+        <li><a href='https://twitter.com/obensource'>Ben Michel</a></li>
+      </ul>
+    </section>
+  </section>
+
+  <section id="temporary">
+    If you're looking for docs, they're available in <a href="https://github.com/nodejs/node/tree/master/doc">nodejs/node</a>. If you're looking for the most recent security release, please see the temporary <a href="https://nodejs.org/en/blog/vulnerability/june-2020-security-releases/">security release landing page</a> in lieu of the downloads page and blog post.
```

Comment:
```suggestion
    If you're looking for docs, they're <a href="https://nodejs.org/docs">still available</a>. If you're looking for information on the most recent security release, please see <a href="https://nodejs.org/en/blog/vulnerability/june-2020-security-releases/">the security release blog post</a>.
```

reviewed the copy and I think this is slightly more accurate. https://nodejs.org/docs will still work. It is a raw folder view of docs for all version... so it is still a less than pleasant experience compared to what we normally offer, but a more complete solution than just pointing to master.

Reply:
LGTM